src/layouts/index.js

```diff
@@ -1,11 +1,11 @@
 import React from "react"
-import Header from "../components/Header/header"
-import Footer from "../components/Footer/footer"
+import Header from "../molecules/Header"
+import Footer from "../molecules/Footer"
 
 export default ({ children }) => (
   <>
-    <Header siteTitle={"Educod@r"} />
-    <main>{children}</main>
+    <Header siteTitle={"Educodar"} />
+    <main style={{ margin: `0 auto`, maxWidth: 768 }}>{children}</main>
```

Comment:
We should not use inline styling here. It's better to create an organism like `<Main>` and create a CSS module file, like `main.module.css` with this content:

```css
.main {
   margin-left: auto;
   margin-right: auto;
   max-width: calc(var(--layout-base)*96);
}
```

And `<Main>` Functional Component would be like this:

```js
({children}) => <main className={styles.main}>{children}</main>
```

Reply:
Done.